### DISCUSSION TITLE
Daily 14/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#137] [General]
Ayer estuve en las clases de SGC y de full-stack donde explicaron más o menos cómo hay que estructurar el proyecto, así que moví los archivos html de sitio a view, agregué los directorios model y controller y metí dentro las clases con comentarios sobre los servicios que en principio aportarían. También ajusté la documentación sobre los controladores en la Wiki. También terminé el diagrama de clases y aproveché para revisar los htmls, arreglar algunos links, ajustar el encabezado para que sean todos iguales, agregué bootstrap a todas las páginas por el encabezado y arreglé varios errores de html.

Hoy a la mañana terminamos con el diagrama relacional para la base de datos y con ellos encontramos varios cambios que hay que hacer al diagrama de clases que voy a arreglar durante el día espero. Y a la noche espero terminar con el diagrama entidad relación partiendo del diagrama de colaboración entre clases. También quiero estar en la clase de full-stack de nuestra aula. Una vez que termine voy a ajustar el readme principal del repositorio para apuntar a la documentación en la wiki con los diagramas y armar un PDF y subirlo a documentación para asegurarme que si pueden clonar el repositorio pueden tener lo de back-end.

Pensando a futuro quiero librarme hoy de todo lo teórico para pasarme todo el sábado y algo del domingo ayudando con html.

**Comment by NataliaAlvarezIspc** [comment]
Ayer corregi los enlaces y agregué los encabezados en la mayoría de los html y los subí al repositorio. Después empecé a darle estilo al index pero no llegué a terminarlo. Estuve ayudando a una compañera a crear ramas en git dependientes de la issues  y a subir archivos al repositorio. Por la tarde noche participé en las clases de SGC y de Full Stack.
Hoy por la mañana estuvimos realizando el diagrama relacional. Continuaré tratando de darle estilo al index y haciendo algunas correcciones.
El fin de semana estaré ayudando en lo que pueda de las tareas que no se hayan hecho.

**Comment by Brian-Almada** [comment]
Ayer estuve con poco tiempo retocando css y algo de html para solucionar css. 
Hoy voy a continuar con eso. Si me queda algo de tiempo voy a ponerme con SGC que no he visto nada del módulo dos aun.
